tools/block-generator/generator/generate.go

```diff
@@ -288,7 +288,8 @@ func (g *generator) WriteBlock(output io.Writer, round uint64) error {
 
 	if round == cachedRound {
 		// one round behind, so write the cached block (if non-empty)
-		fmt.Printf("Received round request %d, but nextRound=%d. Not finishing round.\n", round, nextRound)
+		// Verbose logging
+		//fmt.Printf("Received round request %d, but nextRound=%d. Not finishing round.\n", round, nextRound)
```

Comment:
```suggestion
		if g.vervose {
			fmt.Printf("Received round request %d, but nextRound=%d. Not finishing round.\n", round, nextRound)
		}
```

Reply:
Perfect! I missed this, adding a logging library seemed like overkill.